v03_pipeline/lib/tasks/update_variant_annotations_table_with_new_samples.py

```diff
@@ -48,6 +50,7 @@ class UpdateVariantAnnotationsTableWithNewSamplesTask(BaseVariantAnnotationsTabl
         default=True,
         parsing=luigi.BoolParameter.EXPLICIT_PARSING,
     )
+    allele_registry_password = luigi.Parameter()
```

Comment:
Also, just a heads up we definitely want this to be an env var and not param.  Luigi does support "hidden" params but our old elasticsearch pass was definitely getting passed in plaintext within dataproc.